README.md

```diff
@@ -1,9 +1,9 @@
-<!-- README.md is generated from README.Rmd. Please edit that file -->
-
 # Supply Chain GHG Emission Factors
-
-This repository provides code to produce the [Supply Chain GHG Emission
-Factors](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324).
+Code to produce the most recent version of the [Supply Chain GHG
+Emission Factors](https://cfpub.epa.gov/si/si_public_record_Report.cfm?dirEntryId=349324).
+Code to produce older versions can be found in [releases](/releases),
```

Comment:
the link to releases is breaking

Reply:
Thanks fixed with https://github.com/USEPA/supply-chain-factors/commit/5a2a6fe227e300e20a922cc25a7b37a70cba7486